Contributing.md

```diff
@@ -0,0 +1,36 @@
+
+Welcome to the Databoom Github. This is an open source capstone project focused on recreating the detonation database located at https://shepherd.caltech.edu/detn_db/html/db.html
+
+Please read all of the guidelines mentioned below to ensure consistent coding style, conventions and a well contructed contribution approach.
+
+For any sort of contribution be it documentation or code the contributing method will be the same. If any Pull Requests are submitted and code does not follow outlined method then it should be rejected.
+
+Ground Rules for Development
+
+- Nobody should ever be commiting directly to the master branch. 
+
+    The master branch will be used for production and the site will be hosted off this branch. Master will be updated by pull request following the end of every sprint with the approval of 3 team members. 
+
+- All development will take place on develop branch, this branch is created under the master branch. To update your local git to reflect remote git please use the following command inside your locally cloned repo.
+
+    git fetch --all
+    git checkout develop 
+
+    All feature branches are required to fork off develop branch and upon completion of your feature, you must submit a pull request from your feature branch to merge into develop. To create a branch under develop:
+
+    git checkout -b myfeature_branch develop
+
+- Feature branches should be appended with github issue associated to it. For example, lets say you  are working on a github issue referenced as '[Backend] Add Logic for Sign up #63'. To you should name your branch as 'Backend-SignUp-63'.
```

Comment:
I suggest we put number upfront, eg 63-Backend-SignUp, nubmers will be easier to map in mind

Reply:
> I suggest we put number upfront, eg 63-Backend-SignUp, nubmers will be easier to map in mind

Yeah thats okay. Ill make the modification shortly